boards/arm/rak5010_nrf52840/rak5010_nrf52840.dts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2020 Guillaume Paquet <guillaume.paquet@smile.fr>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "RAKWireless RAK5010 Wistrio Board with a Nordic NRF52840 SoC";
+	compatible = "nordic,rak5010_nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
+			label = "Green LED 0";
+		};
+	};
+
+	/* Decalration of aliases */
```

Comment:
Meaningless comment has a typo :arrow_up:

Reply:
Indeed. I fixed it. Let me know if it is ok for you now ;)